clients/drcachesim/tests/record_filter_unit_tests.cpp

```diff
@@ -427,6 +430,110 @@ test_encodings2regdeps_filter()
     return true;
 }
 
+/* Test SYS_futex preservation.
+ */
+#define SYS_futex_test 202
+#define SYS_fsync_test 74
+static bool
+test_func_marker_filter()
+{
+    constexpr addr_t syscall_base =
```

Comment:
The DR style has all caps for constants; below too.

Reply:
Fixed.